routes/webhook.js

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2017, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+const express = require('express');
+const yaml = require('js-yaml');
+
+const router = express.Router();
+const validPrActions = ['opened', 'reopened', 'synchronize'];
+let webhookSecret = null;
+let curationService;
+
+router.post('/', async (request, response, next) => {
+  const isGithubEvent = request.headers['x-github-event'];
+  if (!isGithubEvent) {
+    return fatal(request, response, 'Not a Github event');
+  }
+
+  // @todo secure webhook, see https://github.com/Microsoft/ghcrawler/blob/develop/routes/webhook.js#L28
+
+  const {pull_request: pr, action: prAction, number} = request.body;
+  const {sha, ref} = pr.head;
```

Comment:
formatting.  spaces around the {} so `{ sha, ref }` rather than `{sha, ref}`.  Here and elsewhere

We should setup linting rules. Actually we have .eslint setup but a) not sure when it is run and b) it does not appear to have all the rules.  Lets iterate on that.

Reply:
Yeah, I had Prettier enabled in my editor on-save, I ended up disabling it. We just started using Prettier as an eslint plugin at work to eliminate style/formatting nitpicks in PRs. We rather like it, but it's not for everyone. https://github.com/prettier/prettier